public/index.html

```diff
@@ -30,323 +30,354 @@ <h1><a class="p-name u-url" href="https://microformats.io/">Microformats</a></h1
-          Microformats can be used for:
-          <a href="http://microformats.org/wiki/h-feed">feeds</a>,
-          <a href="http://microformats.org/wiki/h-entry">blog posts</a>,
-          <a href="http://microformats.org/wiki/h-event">events</a>,
-          <a href="http://microformats.org/wiki/h-adr">locations</a>,
-          <a href="http://microformats.org/wiki/h-review">reviews</a>,
-          <a href="http://microformats.org/wiki/h-recipe">recipes</a>,
-          <a href="http://microformats.org/wiki/h-resume">resumes</a>,
-          and <a href="http://microformats.org/wiki/h-product">products</a>.
-        </p>
-      </section><!-- #what-else -->
-
-
-      <section id="in-the-wild" class="py-5 mb-5">
-        <h2 class="mb-5">Examples in the wild</h2>
-
-        <div class="row">
-          <div class="col-3">
-            <a target="blank" href="https://mastodon.social/about">
-              <img alt="Mastodon" src="/assets/images/examples/mastodon.png" />
-            </a>
-          </div>
-          <div class="col-6">
-            <a target="blank" href="https://indieweb.org">
-              <img alt="IndieWebCamp" src="/assets/images/examples/indie-web-camp.png" />
-            </a>
-          </div>
-          <div class="col-3">
-            <a target="blank" href="https://micro.blog">
-              <img alt="micro.blog" src="/assets/images/examples/micro-blog.png" />
-            </a>
-          </div>
-        </div>
-      </section><!-- #in-the-wild -->
-
-
-      <section id="developers">
-        <h2>Developers</h2>
-
-        <p>
-          Microformats are fully specced with a
-          <a href="https://github.com/microformats/tests">comprehensive test suite</a>
-          and are licensed as Public Domain allowing you to freely use them however you want.
-        </p>
-      </section><!-- #in-the-wild -->
-
-      <section id="parsing">
-        <h2>Parsing Microformats</h2>
-
-        <p>
-          If you&rsquo;re a developer working on an app or service,
-          you probably need JSON as your input format.
-          Don&rsquo;t worry.
-          There are several Microformats parsers written in multiple programming languages.
-          A parser will take a URL or a glob of HTML, understand it,
-          then convert it to JSON for your use.
-        </p>
-
-        <p>You can test your URL here (or HTML snippet on any of the parser sites).</p>
+            </div>
+          </form>
+        </section><!-- #parsing -->
 
+        <section id="parsers">
+          <h2>Parser Libraries</h2>
 
-        <form action="https://ruby.microformats.io/microformats" accept-charset="UTF-8" method="get" class="form -inline mt-3 mb-3">
-          <input name="utf8" type="hidden" value="✓">
+          <p>Production ready parsers are available for: <a href="https://go.microformats.io">Go</a>, <a
+              href="http://node.microformats.io">Node</a>, <a href="https://php.microformats.io">PHP</a>, <a
```

Comment:
Could you make this link https?

Reply:
@gRegorLove since the status of this PR can't be resolved until microformats/microformats-parser-website-node#4 is resolved, I've created a separate PR to fix links: #69.